src/libopensc/cardctl.h

```diff
@@ -305,6 +305,7 @@ enum {
 	SC_CARDCTL_IDPRIME_GET_NEXT_OBJECT,
 	SC_CARDCTL_IDPRIME_FINAL_GET_OBJECTS,
 	SC_CARDCTL_IDPRIME_GET_TOKEN_NAME,
+	SC_CARDCTL_IDPRIME_FINAL_GET_PIN_ID,
```

Comment:
what is the significance of the "final" in this action? I think it should be just `SC_CARDCTL_IDPRIME_GET_PIN_ID`?

Reply:
No significance, only copy-paste-leftover, thanks for spotting.